medusa/version_checker.py

```diff
@@ -306,15 +306,12 @@ def check_for_new_news(self, force=False):
         :force: ignored
         """
         # Grab a copy of the news
-        log.debug(u'check_for_new_news: Checking GitHub for latest news.')
-        try:
-            news = self.session.get(app.NEWS_URL).text
-        except Exception:
-            log.warning(u'check_for_new_news: Could not load news from repo.')
-            news = ''
-
-        if not news:
-            return ''
+        log.debug(u'Checking GitHub for latest news.')
+        response = self.session.get(app.NEWS_URL)
+        news = response.text
```

Comment:
If response is None, youl get an exception on the response.text

Reply:
Will change thanks